pages/community/mission.md

```diff
@@ -34,3 +30,13 @@ to target our activities and actions to serving these three aspects:
   to support the establishment and expansion of RSE positions and
   groups within the research ecosystem including the material for
   making business justifications for RSEs.
+
+## 4. Diversity, Equality, and Inclusion
+
+  We believe Diversity, Equity, and Inclusion (DEI) are core values of the
+  US-RSE Association. A dedicated team of active members continue to lead the
```

Comment:
```suggestion
  US-RSE Association. A dedicated team of active members continues to lead the
```
My impression is that the US tends toward a singular "team" and the UK toward a plural "team" but happy to be corrected.

Reply:
@mtbc I think we are going to edit first in the Google Doc and come back to here? We should check with @exoticDFT and @lparsons